matrix/config.py

```diff
@@ -629,6 +629,24 @@ def __init__(self):
                 ("Reconnect to the server if the lag is greater than this "
                  "value (in seconds)"),
             ),
+            Option(
+                "autoreconnect_delay_growing",
+                "integer",
+                "",
+                1,
+                17,
```

Comment:
17 is a bit of a weird max value, we should probably use the same value the IRC plugin uses.